readme.md

```diff
@@ -14,31 +14,15 @@ A [clamping function](https://en.wikipedia.org/wiki/Clamping_(graphics)) constra
 
 The primary motivation for this function is to bring parity with the [CSS function][css-clamp] of the same name. It will be useful when using it in conjunction with the [CSS Painting API](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Painting_API).
 
-## Examples
+That is, given `Math.clamp(VAL, MIN, MAX)`, it represents exactly the same value as `Math.max(MIN, Math.min(VAL, MAX))`.
 
-Currently the problem is solved in two ways:
-
-- Chaining mathematical operators with if statements or ternary operators
-
-```js
-function clamp(number, minimum, maximum) {
-	if (number < minimum) {
-		return minimum;
-	}
-
-	if (number > maximum) {
-		return maximum;
-	}
-
-	return number;
-}
-```
+## Polyfill
 
 - Nesting [`Math.min`][math-min] and [`Math.max`][math-max] calls
 
 ```js
-function clamp(number, minimum, maximum) {
-	return Math.min(Math.max(number, minimum), maximum);
+function clamp(value, minimum, maximum) {
```

Comment:
Following the implementation, I think it should be `val`, `min` and `max` everywhere. Do they name it `value` in the spec more often?